Snakefile

```diff
@@ -24,18 +24,23 @@ PYPSAEARTH_FOLDER = "pypsa-earth"
 # convert country list according to the desired region
 config["countries"] = create_country_list(config["countries"])
 
+run = config["run"]
 
-SDIR = config["summary_dir"] + config["run"]
-RDIR = config["results_dir"] + config["run"]
+
+SDIR = config["summary_dir"] + run["name"]
+RDIR = config["results_dir"] + run["name"]
 CDIR = config["costs_dir"]
 
 config.update({"git_commit": get_last_commit_message(".")})
 config.update({"submodule_commit": get_last_commit_message(PYPSAEARTH_FOLDER)})
 
+RDIR_PE = run["name"] + "/" if run.get("name") else ""
```

Comment:
Just for understanding: "PE" stands for PyPSA-Earth?

Reply:
Yes. This implementation relies on defining the whole scenario name of pypsa-earth as the one of the -sec model.
Potentially, we could have a different option for that to avoid re-executing the power model for different scenarios of the same power model.
Not sure what is your feeling here

Reply:
Well yes it would be nice to have the option to not rerunning pypsa-earth for every new pypsa-earth-sec scenario indeed. Do you think it is much effort to consider it in this PR? It it is and it would delay the Merge, then we postpone the option to not run pypsa-earth for every new scenario. By the way, isn't there an option called "disable_subworkflow" which disables pypsa-earth? I assume this option would not work anymore in this PR?